### DISCUSSION TITLE
getting 2 means and 2 standard deviations for the same morpho

### DISCUSSION THREAD

**Discussion by asmaaharfoush** [neperfepx/neper#203] [Q&A] [answered]
Hello, 

I am wondering if neper is able to provide 2 means and 2 standard deviations for the same morpho.

In other wording how I can make this distribution has 2 means and 2 standard deviations (neper -T -n from_morpho -morpho "diameq:lognormal(0.1,0.03)")

Thanks!

**Comment by rquey** [comment, marked as answer]
Just add the distributions, as in ` -morpho "diameq:lognormal(0.1,0.03)+lognormal(0.2,0.06)"`. You can also weight them (in terms of grain number), as in `-morpho "diameq:0.8*lognormal(0.1,0.03)+0.2*lognormal(0.2,0.06)"`.

(You should also preferably prescribe a sphericity distribution.)

**Reply by asmaaharfoush** [reply]
That is really helpful, thank you!
